<launch>

  <node pkg="lslidar_m10_driver" type="lslidar_m10_driver_node" name="lslidar_m10_driver_node" output="screen">
    <param name="frame_id" value="laser_link"/>
    <param name="device_ip" value="192.168.1.200"/>
    <param name="device_port" value="2368"/>
  </node>

  <node pkg="lslidar_m10_decoder" type="lslidar_m10_decoder_node" name="lslidar_m10_decoder_node" output="screen">
    <param name="child_frame_id" value="laser_link"/>
	<param name="scan_topic" value="scan"/>
    <param name="point_num" value="948"/>
    <param name="angle_disable_min" value="0.0"/>
	<param name="angle_disable_max" value="0.0"/>
    <param name="min_range" value="0.3"/>
    <param name="max_range" value="100.0"/>
  </node>

</launch>
